io.github.fastrizwaan.WebkitWord.yaml

```diff
@@ -0,0 +1,27 @@
+id: io.github.fastrizwaan.WebkitWord
+sdk: org.gnome.Sdk
+runtime: org.gnome.Platform
+runtime-version: '48'
+command: webkitword
+base: org.libreoffice.LibreOffice
+base-version: stable
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=pulseaudio
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --talk-name=org.a11y.Bus
+  - --filesystem=xdg-run/gvfs
+  - --filesystem=xdg-run/gvfsd
```

Comment:
you shouldn't need both for gtk apps https://docs.flatpak.org/en/latest/sandbox-permissions.html#gvfs-access

Reply:
using only xdg-run/gvfs